packages/core/comparison/lib/isIntentionalDifference/index.ts

```diff
@@ -47,29 +49,35 @@ const filters = [
 const isIntentionalDifference = (
   expected: AnnotatedHearingOutcome,
   actual: AnnotatedHearingOutcome,
-  incoming: AnnotatedHearingOutcome
+  incomingMessage: AnnotatedHearingOutcome | PncUpdateDataset
 ): boolean => {
-  const expectedMatch = summariseMatching(expected, true)
-  const actualMatch = summariseMatching(actual, true)
+  const intentionalDifference: IntentionalDifference = {
```

Comment:
Naming-wise, this isn't an intentional difference, it's the comparison data we're using to determine if something is an intentional difference. So I wonder about naming this type something like `ComparisonData` to represent the intention?

Reply:
Resolved in https://github.com/ministryofjustice/bichard7-next-core/pull/795/commits/6b449ed90988d66085f2346d362df6bd24d8e8ea.